!SESSION 2024-10-16 00:03:37.743 -----------------------------------------------
eclipse.buildId=4.5.2.M20160212-1500
java.version=1.8.0_202
java.vendor=Oracle Corporation
BootLoader constants: OS=win32, ARCH=x86_64, WS=win32, NL=en_US
Framework arguments:  -product org.eclipse.epp.package.jee.product
Command-line arguments:  -os win32 -ws win32 -arch x86_64 -product org.eclipse.epp.package.jee.product

!ENTRY org.eclipse.epp.logging.aeri.ide 2 16 2024-10-16 00:08:31.112
!MESSAGE Server ‘org.eclipse.epp.logging.aeri.ide.server’ failed with exception: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was STRING at line 1 column 1 path $
See https://github.com/google/gson/blob/main/Troubleshooting.md#unexpected-json-structure. ; version: 1.100.0.v20160217-0435
!STACK 0
com.google.gson.JsonSyntaxException: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was STRING at line 1 column 1 path $
See https://github.com/google/gson/blob/main/Troubleshooting.md#unexpected-json-structure
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:520)
	at com.google.gson.Gson.fromJson(Gson.java:1361)
	at com.google.gson.Gson.fromJson(Gson.java:1262)
	at com.google.gson.Gson.fromJson(Gson.java:1171)
	at com.google.gson.Gson.fromJson(Gson.java:1137)
	at org.eclipse.epp.internal.logging.aeri.ide.server.json.Json.deserialize(Json.java:88)
	at org.eclipse.epp.internal.logging.aeri.ide.server.mars.IO.refreshConfiguration(IO.java:70)
	at org.eclipse.epp.internal.logging.aeri.ide.server.mars.ServerConnection.startUp(ServerConnection.java:101)
	at com.google.common.util.concurrent.AbstractIdleService$2$1.run(AbstractIdleService.java:54)
	at com.google.common.util.concurrent.Callables$3.run(Callables.java:93)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was STRING at line 1 column 1 path $
See https://github.com/google/gson/blob/main/Troubleshooting.md#unexpected-json-structure
	at com.google.gson.stream.JsonReader.unexpectedTokenError(JsonReader.java:1768)
	at com.google.gson.stream.JsonReader.beginObject(JsonReader.java:469)
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:509)
	... 10 more

!ENTRY org.eclipse.egit.ui 2 0 2024-10-16 00:08:43.199
!MESSAGE Warning: The environment variable HOME is not set. The following directory will be used to store the Git
user global configuration and to define the default location to store repositories: 'C:\Users\HP'. If this is
not correct please set the HOME environment variable and restart Eclipse. Otherwise Git for Windows and
EGit might behave differently since they see different configuration options.
This warning can be switched off on the Team > Git > Confirmations and Warnings preference page.

!ENTRY org.eclipse.egit.ui 4 0 2024-10-16 00:12:35.050
!MESSAGE Exception caught during execution of add command
!STACK 0
org.eclipse.jgit.api.errors.JGitInternalException: Exception caught during execution of add command
	at org.eclipse.jgit.api.AddCommand.call(AddCommand.java:211)
	at org.eclipse.egit.core.op.CommitOperation.addUntracked(CommitOperation.java:224)
	at org.eclipse.egit.core.op.CommitOperation.access$4(CommitOperation.java:212)
	at org.eclipse.egit.core.op.CommitOperation$1.run(CommitOperation.java:197)
	at org.eclipse.core.internal.resources.Workspace.run(Workspace.java:2241)
	at org.eclipse.egit.core.op.CommitOperation.execute(CommitOperation.java:208)
	at org.eclipse.egit.ui.internal.commit.CommitJob.run(CommitJob.java:98)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
Caused by: java.io.IOException: The process cannot access the file because another process has locked a portion of the file
	at java.io.FileInputStream.readBytes(Native Method)
	at java.io.FileInputStream.read(Unknown Source)
	at org.eclipse.jgit.util.IO.readFully(IO.java:303)
	at org.eclipse.jgit.internal.storage.file.ObjectDirectoryInserter.insert(ObjectDirectoryInserter.java:103)
	at org.eclipse.jgit.api.AddCommand.call(AddCommand.java:187)
	... 7 more

!ENTRY org.eclipse.egit.core 4 0 2024-10-16 00:13:01.221
!MESSAGE An exception occurred during push on URI https://github.com/GurunathamCH/Final.git: An internal Exception occurred during push: Source ref refs/heads/main doesn't resolve to any object.
!STACK 0
org.eclipse.jgit.api.errors.JGitInternalException: Exception caught during execution of push command
	at org.eclipse.jgit.api.PushCommand.call(PushCommand.java:182)
	at org.eclipse.egit.core.op.PushOperation.run(PushOperation.java:228)
	at org.eclipse.egit.ui.internal.push.PushOperationUI.execute(PushOperationUI.java:167)
	at org.eclipse.egit.ui.internal.push.PushOperationUI$1.run(PushOperationUI.java:229)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
Caused by: java.io.IOException: Source ref refs/heads/main doesn't resolve to any object.
	at org.eclipse.jgit.transport.RemoteRefUpdate.<init>(RemoteRefUpdate.java:288)
	at org.eclipse.jgit.transport.RemoteRefUpdate.<init>(RemoteRefUpdate.java:192)
	at org.eclipse.jgit.transport.Transport.findRemoteRefUpdatesFor(Transport.java:646)
	at org.eclipse.jgit.transport.Transport.findRemoteRefUpdatesFor(Transport.java:1269)
	at org.eclipse.jgit.api.PushCommand.call(PushCommand.java:153)
	... 4 more
